### ISSUE TITLE
When I use the workflow 'cogvideox_Fun_180_orbit_01', I encounter some problems. Could you please help?

### ISSUE THREAD

**Issue by Jesssssss123** [kijai/ComfyUI-CogVideoXWrapper#297]
![image](https://github.com/user-attachments/assets/d763fd68-bd39-4ef0-bcb4-1025cb49809d)
![image](https://github.com/user-attachments/assets/68833b08-873b-480b-9e32-0d36c77ffa76)

**Comment by Jesssssss123** [comment]
All the parameter settings are default, and both the model and requirements have been downloaded.

**Comment by KrakeyMTL** [comment]
I think orbit you need to change the sizing I never used it - normally I only get those tensor match errors when i screwed up the resolutions it wants. Like 1.5b T2V does 1360x768 but if I forget and I'm on the I2V model that takes any rez it will give a similar error.

I only ever see these when i mess that up - maybe same for you?  Also the orbit is the 360 around the image right? not sure on the settings for that as I've never used it!  

My 2 cents to try and help lol

**Comment by GeffersAtWork** [comment]
Yeah I think it's do with the sizes. I remember there was something weird about the workflow in that it seemed to do everything twice, so I removed the second part which didn't seem to do anything.  I'll give you my settings which worked: start image supplied, without end image. Scaled up to 720x480. Tiling enabled aug strength 0.03, num frames input to 33. T5 google clip, orbit left model into lora for cogvideo loader at strength 1.0. Quantization fp8, attention mode Comfy,  load device main device,  cpu offload = false. using model alibaba fun 1.5 inP. Tiling doing 360x240 overlap 0.20

**Comment by kijai** [comment]
It was pretty hastily put together workflow, organized it some to make it more clear, it does everything twice because it first orbits right, then left and then joins the clips:

https://github.com/user-attachments/assets/da61b1bf-d0a3-43f8-b47b-d516c0602664

As to the size mismatch, that was potentially caused by the input being resized with "keep proportions" on so some input sizes would become incompatible, I forgot to set the "divisible_by" to 16  which should prevent that, if not then disable "keep proportions" and use crop instead.

**Comment by KrakeyMTL** [comment]
@kijai all this requires more sleep bud, a luxury once you are in this rabbit hole ;) haha
